src/Common/CurrentMetrics.cpp

```diff
@@ -70,6 +70,7 @@
     M(ZooKeeperSession, "Number of sessions (connections) to ZooKeeper. Should be no more than one, because using more than one connection to ZooKeeper may lead to bugs due to lack of linearizability (stale reads) that ZooKeeper consistency model allows.") \
     M(ZooKeeperWatch, "Number of watches (event subscriptions) in ZooKeeper.") \
     M(ZooKeeperRequest, "Number of requests to ZooKeeper in fly.") \
+    M(ZooKeeperExpiredSessions, "Number of expired ZooKeeper sessions. Includes only non-auxiliary sessions managed by the context.") \
```

Comment:
Why only for non-auxiliary? `sessions managed by the context` looks strange, like attachment to place in the code.

Actually, I don't remember already what the difference between auxiliary and non-auxiliary sessions. Please, could you clarify?